src/main/java/com/github/sbt/avro/mojo/AvscFilesCompiler.java

```diff
@@ -122,7 +123,7 @@ private boolean tryCompile(AvroFileRef src, File outputDirectory) {
     try {
       schema = schemaParser.parse(src.getFile());
       validateParsedSchema(src, schema);
-    } catch (SchemaParseException e) {
+    } catch (SchemaParseException | AvroTypeException e) {
```

Comment:
Can you give some context on the added AvroTypeException? I didn't see it added to the SchemaParser#parse signature in 1.12

Reply:
Parser logic changed. We used to have
```
 org.apache.avro.SchemaParseException: "com.github.sbt.avro.test.C" is not a defined name. The type of the "c" field must be a defined name or a {"type": ...} expression.
```
Now in `1.12`
```
org.apache.avro.AvroTypeException: Undefined schema: com.github.sbt.avro.test.C
```

Reply:
Both are `AvroRuntimeException`, it's probably better to use this class instead here